### ISSUE TITLE
Exoplayer Timebar scrub with Label

### ISSUE THREAD

**Issue by brunocaiado** [google/ExoPlayer#7854]
Hey!

I have been developing an app using Exoplayer and I am in needs to put a label on the TimeBar Scrub. 

I saw that Youtube has the Player Time Position moving the Scrub like we can see in the next image:

![image](https://user-images.githubusercontent.com/29310099/92247185-c2588700-eebe-11ea-920a-bf11c78c7392.png)

In parallel, I find out that Material Design Slider has something similar too. 
https://material.io/components/sliders#anatomy - **_3. Value label (optional)_**

Maybe it would be nice that Exoplayer could have an easy implementation of this?

Thanks!

**Comment by christosts** [comment]
@ojw28 to prioritise this in the backlog

**Comment by ojw28** [comment]
Given the current scrub position is already shown to the left hand side, it's unclear what purpose a label on the scrub position itself would serve. Please could you clarify?

Note that thumbnails during scrubbing, such as those in YouTube, are typically provided out-of-band (e.g., as tiled images of thumbnails that the client can request separately to the main video) and in a way that's custom to the app.

**Comment by brunocaiado** [comment]
I'm sorry about the delayed response. 

@ojw28 The purpose is just a product issue. Maybe there is more people needing this feature like I needed, and then I just asked to see what you guys think about it. Just because, maybe it was a good ideia do include on ExoPlayer for other people easily use it.

In the meanwhile I did an implementation based on https://github.com/rubensousa/PreviewSeekBar, where moving the scrub there is a label on the top of it, displaying the scrub position. About the thumbnail, I understand that it is something out of the scope of the ExoPlayer, since it is another topic. 

Another issue that I had around this was having another label displaying the "current played time" ( in the attached SS is the one "00:02" ). For this player behavior, what is the best approach for the "current played time" label?

**Comment by ojw28** [comment]
I'm going to close this, since it seems rather specific to your particular requirements, and you've also managed to implement it yourself. I don't think there's a strong argument for adding this feature directly into the library.

It is of course possible (and likely) that at least someone else requires similar functionality, but it's infeasible for us to support all possible UI features that more than one developer needs. Our UI components aim to provide a starting point for simple use cases, along with a basic amount of customization. Where a developer has a specific UX requirement they'll normally need to implement their own UI components for playback, which is what nearly all major streaming apps do.
